{"name": "test_unsuccessfull_create_courier_with_the_same_data", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  \n  \u001b[0m\u001b[91m- Этот логин уже используется\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Этот логин уже используется. Попробуйте другой.\u001b[39;49;00m\u001b[90m\u001b[39;49;00m", "trace": "self = <tests.test_create_courier.TestCreateCoirier object at 0x0000025960881E50>\n\n    def test_unsuccessfull_create_courier_with_the_same_data(self):\n            with allure.step(\"генерируем логин, пароль и имя курьера\"):\n                login = generate_random_string(10)\n                password = generate_random_string(10)\n                first_name = generate_random_string(10)\n    \n            with allure.step(\"формирование тела запроса\"):\n                payload = {\n                    \"login\": login,\n                    \"password\": password,\n                    \"firstName\": first_name\n                }\n    \n            with allure.step(\"отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response\"):\n                response = requests.post(Path.CREATE_COURIER, data=payload)\n            with allure.step(\"проверка кода ответа\"):\n                assert response.status_code == 201\n            with allure.step(\"отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response_with_the_same_data\"):\n                response_with_the_same_data = requests.post('https://qa-scooter.praktikum-services.ru/api/v1/courier', data=payload)\n            with allure.step(\"проверка кода и тела ответа\"):\n                assert response_with_the_same_data.status_code == 409\n>               assert response_with_the_same_data.json()[\"message\"] == \"Этот логин уже используется\"\nE               AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\nE                 \nE                 \u001b[0m\u001b[91m- Этот логин уже используется\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\nE                 \u001b[92m+ Этот логин уже используется. Попробуйте другой.\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n\ntests\\test_create_courier.py:51: AssertionError"}, "steps": [{"name": "генерируем логин, пароль и имя курьера", "status": "passed", "start": 1757337439379, "stop": 1757337439379}, {"name": "формирование тела запроса", "status": "passed", "start": 1757337439379, "stop": 1757337439379}, {"name": "отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response", "status": "passed", "start": 1757337439379, "stop": 1757337440002}, {"name": "проверка кода ответа", "status": "passed", "start": 1757337440002, "stop": 1757337440002}, {"name": "отправляем запрос на регистрацию курьера и сохраняем ответ в переменную response_with_the_same_data", "status": "passed", "start": 1757337440002, "stop": 1757337440361}, {"name": "проверка кода и тела ответа", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  \n  \u001b[0m\u001b[91m- Этот логин уже используется\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n  \u001b[92m+ Этот логин уже используется. Попробуйте другой.\u001b[39;49;00m\u001b[90m\u001b[39;49;00m\n", "trace": "  File \"c:\\Users\\Dmitry\\Desktop\\Python YP\\Sprint_7\\tests\\test_create_courier.py\", line 51, in test_unsuccessfull_create_courier_with_the_same_data\n    assert response_with_the_same_data.json()[\"message\"] == \"Этот логин уже используется\"\n"}, "start": 1757337440361, "stop": 1757337440364}], "start": 1757337439378, "stop": 1757337440366, "uuid": "e1748b5a-dc71-4f03-a64c-c4a9db2f1689", "historyId": "30a2afc4c8f1dc1ae853d63f7c8cd31a", "testCaseId": "30a2afc4c8f1dc1ae853d63f7c8cd31a", "fullName": "tests.test_create_courier.TestCreateCoirier#test_unsuccessfull_create_courier_with_the_same_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCoirier"}, {"name": "host", "value": "DESKTOP-A445VSF"}, {"name": "thread", "value": "14248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}], "titlePath": ["tests", "test_create_courier.py", "TestCreateCoirier"]}